source/presales_evaluation/.ipynb_checkpoints/presales_evaluation-checkpoint.ipynb

```diff
@@ -730,7 +736,9 @@
     "- evaluate region confidence with SME max\n",
     "- evaluate region confidence with SME min\n",
     "\n",
-    "We use a Gaussian Mixture Model (GMM) to fit to the data to be able to cluster distributions with means and covariances. We can then sample the GMM with a random state-action pair and evaluate the regions to trust based compared to SME desired limits. "
+    "We use a Gaussian Mixture Model (GMM) to fit to the data to be able to cluster distributions with means and covariances. We can then sample the GMM with a random state-action pair and evaluate the regions to trust based compared to SME desired limits. \n",
```

Comment:
Similar to presales_evaluation.ipynb, I'd suggest test/evaluate/run instead of sample.